### PR TITLE
animations added to fragment transaction for better performance

### PR DESCRIPTION
**Pull request by amindadgar**

### CODE REVIEW

File: app/src/main/java/com/anant/codingguide/activity/ControlActivity.kt [hunk 1]
```diff
@@ -71,23 +71,29 @@ class ControlActivity : AppCompatActivity() {
                 R.id.language->{
                     supportFragmentManager.beginTransaction().replace(R.id.frame,
                         LanguagesFragment()
-                    ).commit()
+                    )
+                        .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
```
Comment (anantcodes) [inline app/src/main/java/com/anant/codingguide/activity/ControlActivity.kt:75]: Kindly use some other animation.

File: app/src/main/java/com/anant/codingguide/activity/ControlActivity.kt [hunk 2]
```diff
@@ -71,23 +71,29 @@ class ControlActivity : AppCompatActivity() {
...
                     supportActionBar?.title="Programming languages"
                     drawerLayout.closeDrawers()
                 }
 
                 R.id.dev->{
                     supportFragmentManager.beginTransaction().replace(R.id.frame,
                         DevelopmentFragment()
-                    ).commit()
+                    )
+                        .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
```
Comment (anantcodes) [inline app/src/main/java/com/anant/codingguide/activity/ControlActivity.kt:85]: Kindly use some other animation.

**Review by anantcodes — COMMENTED** [review]
Use a different animation.
Thank you

**Comment by anantcodes** [comment]
@amindadgar Kindly send different PR for different issues to maintain the ease in code review process.

**Comment by amindadgar** [comment]
sorry for my late answer, I didn't see the notification.
I'll do it in a day.

**Pull request closed without merging**